README.md

```diff
@@ -1,9 +1,10 @@
-#Ollivanders2
-##What is Ollivanders2?
+# Ollivanders2
```

Comment:
Did you intend to reformat the readme?

Reply:
Yes, the current formatting doesn't appear properly on GitHub. It shows the hashtags in the titles.

https://imgur.com/a/Q6MpuVj
This is what it looks like without, while I am on mobile it's the same on my PC. Adding spaces after the # fixes the markdown.

Reply:
Well, I forgot we even had a readme so there ya go. :)